src/root.component.tsx

```diff
@@ -161,3 +161,37 @@ export default openmrsRootDecorator({
           <Sidebar></Sidebar>
         </div>
 */
+
+export type ChartConfigType = {
```

Comment:
I don't think it's the convention out there to postfix with 'Type'. It ends up being verbose too. It's just like prefixing an Interface with I is discouraged. 'ChartConfig' is sufficient, because one can tell that this is a config.

Reply:
Sounds good. We will use PascalCase

Reply:
I have another PR based on this branch. Let me correct it in the next PR as otherwise I'll have to do it twice.